wled00/FX_fcn.cpp

```diff
@@ -837,13 +866,101 @@ uint32_t Segment::getPixelColor(int i)
   }
 #endif
 
+  return strip.getPixelColor(getPixelIndex(i));
+}
+
+uint32_t Segment::getBufferColor(int i) {
```

Comment:
Unnecessary if you'd be using `Segment::_buffer[]`.

Reply:
I would still have to calculate the pixel index, right? So it would be `Segment::_buffer[getPixelIndex(i)]`. I tried to implement this as an inline function, but the compiler yelled at me. Happy to revisit this or make it a macro.